.travis.yml

```diff
@@ -8,25 +8,34 @@ matrix:
       env:
         - PYTHON=3.6
         - PYTHONWARNINGS=ignore
+        - RAY_DEFAULT_BUILD=1
+        - RAY_CYTHON_EXAMPLES=1
+      before_install:
+        - . ./ci/travis/ci.sh prepare RAY_CI_SERVE_AFFECTED,RAY_CI_TUNE_AFFECTED,RAY_CI_PYTHON_AFFECTED
```

Comment:
I can if you really prefer that, but I was hoping trying not to add more environment variables, since they make the code really rigid and and have already been giving me a hard time.